src/providers/workers-ai/complete.ts

```diff
@@ -0,0 +1,122 @@
+import { Params } from '../../types/requestBody';
+import { CompletionResponse, ErrorResponse, ProviderConfig } from '../types';
+import { WORKERS_AI } from '../../globals';
+import { generateErrorResponse, generateInvalidProviderResponseError } from '../utils';
+
+export const WorkersAiCompleteConfig: ProviderConfig = {
```

Comment:
WorkersAI supports some other params as well which can be mapped here and in chatComplete config as well. The params which do not have a mapping with existing OpenAI param can be added as new params in this config. 

Suggesting this addition to the config:
```
raw: {
    param: 'raw'
  },
  max_tokens: {
    param: "max_tokens"
  }
```

Reply:
@VisargD added in https://github.com/Portkey-AI/gateway/pull/269/commits/18510fff26e04b59c57f49ae613d7bac7934bba9